src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -375,13 +376,13 @@ private async Task<List<McPkg>> GetMcPkgsToAddAsync(IList<string> mcPkgScope, st
                 throw new IpoValidationException("Could not find all mc pkgs in scope.");
             }
 
-            var initialMcPkg = mcPkgDetailsList.FirstOrDefault();
-            if (initialMcPkg != null)
+            var initialCommPkg = mcPkgDetailsList.FirstOrDefault();
```

Comment:
why this rename, for me initialMcPkg is better since picking the first from mcPkgDetailsList

Reply:
sorry just a copypaste mistake